### ISSUE TITLE
Why are /Agrave/agrave/Eacute/eacute needed for German Language support?

### ISSUE THREAD

**Issue by opr-inzn** [mekkablue/Glyphs-Scripts#302]
In the "Test Languages" script – having generated ÄÖÜ, their corresponding lowercases as well as ß – I get the message that German is not supported `German (120000000 speakers): /Agrave/agrave/Eacute/eacute`. To my knowledge and also the Alphabet Charset Builder (http://www.alphabet-type.com/tools/charset-builder/), there is no need to have those in a German character set?

**Comment by mekkablue** [comment]
I do not agree. This is a correct German sentence: »Im Café haben wir zwei Bier à fünf Euro konsumiert.« 

The discussion about what is necessary for supporting a language, and what is optional, is quite complex. If you do not have é and à, most German novels cannot be printed. The cap sharp S is easily replaced, but é and à are not.
